.gitignore

```diff
@@ -16,3 +16,6 @@ spec/reports
 test/tmp
 test/version_tmp
 tmp
+.ruby-env
```

Comment:
Can you remove the `.ruby-gemset` and `.ruby-version` from the `.gitignore` and check those files it.  It would be nice to have sane defaults for these for rvm users that would like to contribute.  Sorry if that wasn't clear before.